src/main/java/org/spburegistry/backend/entity/Project.java

```diff
@@ -0,0 +1,76 @@
+package org.spburegistry.backend.entity;
+
+import jakarta.annotation.Nullable;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.Set;
+
+@EqualsAndHashCode(callSuper = false)
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name="projects")
+@Data
+public class Project extends BaseEntity {
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private String description;
+
+    @ManyToOne
+    @JoinColumn(name = "studying_program_id", nullable = false)
+    private EducationalProgram educationalProgram;
+
+    @Column(name = "repository_link")
+    @Nullable
+    private String repositoryLink;
```

Comment:
May be here we need more than one link, for example:
- for git
- for asana
- for notion

Reply:
Fixed.